app/views/shared/_nav_bar.html.haml

```diff
@@ -6,7 +6,20 @@
 
   %ul#user_menu.navbar-nav.px-3
     %li
-      %span.oi.oi-person{"aria-hidden" => "true", :title => "person"}
+      .btn-group
+        %button.btn.btn-secondary.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown", :type => "button"}
+          = t(:"titles.language")
+        .dropdown-menu.dropdown-menu-right
+          %a
+            - if current_user.locale_en?
```

Comment:
Here we don’t need a conditional. Just have a list of locales to choose from. I.e. English and Khmer. When they clock the locale it switches to that language.

Also I’m not sure if we need the flags. Maybe we can have the drop down menu as the globe icon in font awesome and just put the words English and ខ្មែរ​